epub33/fxl-a11y/index.html

```diff
@@ -347,6 +378,40 @@ <h4>Non-text content or images of text</h4>
                     <li>Purpose, does the content need to be an image or video or could it be better integrated in another way</li>
                 </ul>
             </section>
+
+            <section id="legibility-fonts">
+                <h4>Font selection</h4>
+
+                <p>There is no single font that meets the legibility needs of all users, but considering certain font characteristics to increase legibility is possible. When planning font selections in fixed layout publications, consider the following:</p>
+
+                <ul>
+                    <li>Font sizing</li>
+                    <li>Font weight</li>
+                    <li>Font face or style</li>
+                </ul>
+
+                <section id="legibility-fonts-sizing">
+                    <h5>Font sizing</h5>
+
+                    <p>There is no font size guideline in WCAG [[wcag2]], however the standard default font size in most desktop and mobile browsers is 16pt for body text (i.e. in a <code>p</code> element). This size is sufficient for most content, and headings should be based off of it by using <code>em</code> or <code>rem</code> sizing in CSS [[css2]]. If content contains a great deal of text, it is also recommended to consider a larger body font size like 18pt to ensure readability.</p>
```

Comment:
I'm not sure it's accurate to say there isn't a font size requirement. 1.4.4 says that text needs to be able to resize to 200% of its default size without loss of readability or functionality.

Reading systems that run on tablets or similar devices should let you zoom fxl pages, but desktop-based apps don't, at least not when I last checked (which was some time back). So long as the text scales with the page on the RSes that do zoom, this SC won't be an issue. You can expect that users who need the functionality will seek out those RSes. But if reading systems don't scale the text, and you can't modify the text properties as a user, then there'd be a problem.

(This also leads into the ever problematic reflow SC where zooming fxls doesn't reflow the text as it enlarges, so you end up having to scroll to read lines.)

Reply:
And to be clear, I'm not expecting this to be changed before you merge. Just wanted to mention it.

Reply:
The default font size in browsers is 16px, not 16pt, as far as I know.

Another thing I thought about:

If the RS allows scaling the page, the text size in a fixed layout book will be influenced by two things, when read on a smaller screen:

1. font size setting
1. page size and/or line length

The larger the page, the smaller the text will be when the page is scaled down to fit the viewport. If the lines flow out of the visible area, so that you have to scroll horizontally, the text will be very difficult to read.